ci/scripts/driver_weekly.sh

```diff
@@ -0,0 +1,106 @@
+# and create a pr using GitHub CLI by moving and replacing the yaml case files in
+# ${HOMEgfs}/ci/cases/weekly to {HOMEgfs}/ci/cases/pr.  Then the requisite labels are added
+# so that the current BASH CI framework can then run these cases.  Since this script
+# creates a PR with the CI-Ready labels, the BASH CI framework will automatically run these cases 
+# from that point so it is only required to run this script once from a single machine.
+##############################################################################################
+
+#################################################################
+# TODO using static build for GitHub CLI until fixed in HPC-Stack
+#################################################################
+export GH=${HOME}/bin/gh
+export REPO_URL=${REPO_URL:-"https://github.com/NOAA-EMC/global-workflow.git"}
+
+################################################################
+# Setup the relative paths to scripts and PS4 for better logging
+################################################################
+HOMEgfs="$(cd "$(dirname  "${BASH_SOURCE[0]}")/../.." >/dev/null 2>&1 && pwd )"
+scriptname=$(basename "${BASH_SOURCE[0]}")
+echo "Begin ${scriptname} at $(date -u)" || true
+export PS4='+ $(basename ${BASH_SOURCE[0]})[${LINENO}]'
+
+#########################################################################
+# Set up runtime environment variables for accounts on supported machines
+#########################################################################
+
+source "${HOMEgfs}/ush/detect_machine.sh"
+case ${MACHINE_ID} in
+  hera | orion)
+    echo "Running Automated Testing on ${MACHINE_ID}"
+    source "${HOMEgfs}/ci/platforms/${MACHINE_ID}.sh"
+    ;;
+  *)
+    echo "Unsupported platform. Exiting with error."
+    exit 1
+    ;;
+esac
+
+######################################################
+# setup runtime env for correct python install and git
+######################################################
+set +x
+source "${HOMEgfs}/ush/module-setup.sh"
+module use "${HOMEgfs}/modulefiles"
+module load "module_gwsetup.${MACHINE_ID}"
+set -x
+
+#########################################################
+# Create a new branch from develop and move yaml files
+#########################################################
+branch="weekly_ci_$(date +%Y%m%d)"
+develop_dir="${GFS_CI_ROOT}/develop_temp"
+HOMEgfs_PR="${develop_dir}/global-workflow"
+echo "Creating new branch ${branch} from develop on ${MACHINE_ID} in ${develop_dir}"
+rm -Rf "${develop_dir}"
+mkdir -p "${develop_dir}"
+cd "${develop_dir}" || exit 1
+git clone "${REPO_URL}"
+cd "${HOMEgfs_PR}" || exit 1
+git checkout -b "${branch}"
+
+######################################################
+# move yaml files from ci/cases/weekly to ci/cases/pr 
+# and push new branch for PR weekly CI tests to GitHub
+
+rm -Rf "${HOMEgfs_PR}/ci/cases/pr"
+mv "${HOMEgfs_PR}/ci/cases/weekly" "${HOMEgfs_PR}/ci/cases/pr"
+cd "${HOMEgfs_PR}" || exit 1
```

Comment:
I think we are already in `${HOMEgfs_PR}`.  There is no need to cd into it again.